src/Orleans.Core.Abstractions/CodeGeneration/InvokeMethodRequest.cs

```diff
@@ -67,7 +67,8 @@ public enum InvokeMethodOptions
 
         // Transactional method options. 
         // NOTE: keep in sync with TransactionOption enum.
-        TransactionRequired = 0x200,
-        TransactionRequiresNew = 0x400,
+        TransactionNotSupported = 0x200,
+        TransactionRequired = 0x400,
+        TransactionRequiresNew = 0x800,
```

Comment:
These are implemented like flags, but using them like flags could be contradictory.
Suggest `NotSupported` be the implied result of no value being set and leaving the existing values as-is.
Alternatively, if the default is "supported", then we can define `NotSupported = 0x600` so at least there's a meaning to that value and they're not flags (plus we save a precious bit 😉).

What do you think?